.github/actions/k8s-auth/action.yml

```diff
@@ -7,9 +7,6 @@ inputs:
   cloud_provider:
     description: "Which cloud service provider to use - Google Cloud: 'gcp' or Azure: 'az'"
     default: "gcp"
-  github_token:
-    description: "GitHub token"
-    required: true
```

Comment:
This will not work, must be accepted as input. Imagine if a random action could just willy-nilly access all our secrets without being granted explicit access... 🤠

Reply:
I should have highlighted https://github.com/entur/gha-meta/pull/34/files#diff-10ca432c426ea26768f0ba50c81305f128d1a267f5a0dfa8339f0d5ff7b678d8R53 `GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}` as this is the part that won't work

TLDR: we need this input

Reply:
This does not compute.. at all :D 
We are basically doing the same as here: https://github.com/weinong/azure-federated-identity-samples/blob/main/.github/workflows/access-aks.yml
We don't need to forward the github_token at all.

Reply:
https://azure.github.io/kubelogin/concepts/login-modes/workloadidentity.html

Reply:
https://github.com/Azure/use-kubelogin?tab=readme-ov-file#kubelogin-version

We are _not_ doing the same as the workflows you linked, because we go at it via an action, which has no access to the token as opposed to the workflow.

Reply:
We should probably add tests for these actions in `ci.yml` and if we get a ✅ without GITHUB_TOKEN then I'm OK with the `use-kubelogin` documentation being wrong